package.json

```diff
@@ -4,35 +4,24 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
-    "@types/jest": "^24.0.0",
-    "@types/node": "^12.0.0",
-    "@types/react": "^16.9.0",
-    "@types/react-dom": "^16.9.0",
     "axios": "^0.19.2",
     "bootstrap": "^4.5.0",
     "bs-custom-file-input": "^1.3.4",
     "gh-pages": "^3.1.0",
```

Comment:
as far as I can tell, we're not using GitHub pages for any deployment process, so we may be able to remove this